projects/npm-tools/packages/npm-scripts/src/utils/tweakWebpackConfig.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const path = require('path');
+const {
+	container: {ModuleFederationPlugin},
+} = require('webpack');
+
+const createTempFile = require('./createTempFile');
+const getMergedConfig = require('./getMergedConfig');
+const parseBnd = require('./parseBnd');
+
+const CORE_REMOTES = ['frontend-js-react-web'];
+const CORE_SHARES = ['react'];
+
+const BABEL_CONFIG = getMergedConfig('babel');
+
+function tweakWebpackConfig(webpackConfig, {federation} = {}) {
+	const ret = [];
```

Comment:
In the guidelines we ask that people [avoid abbreviations](https://github.com/liferay/liferay-frontend-guidelines/blob/master/general/naming.md#avoid-abbreviations).